src/expressions/antlr-interface/Expr.g4

```diff
@@ -30,6 +30,7 @@ expr
     | IDENTIFIER '(' expr ')'                  # function
     | IDENTIFIER '[' shift ']'                 # timeShift
     | IDENTIFIER '[' expr  ']'                 # timeIndex
+    | '(' expr ')' '[' shift ']'               # timeShiftExpr
```

Comment:
Reminder : update the grammar in https://github.com/AntaresSimulatorTeam/andromede-modeling-prototype

Reply:
https://github.com/AntaresSimulatorTeam/andromede-modeling-prototype/pull/68